Makefile

```diff
@@ -26,7 +26,7 @@ install:
 
 vendor-update:
 	@echo Updating vendored packages
-	@GO15VENDOREXPERIMENT=1 glide install
+	@GO15VENDOREXPERIMENT=1 glide up
```

Comment:
This has been a bug with glide-0.8.2. Refer https://github.com/Masterminds/glide/issues/185 . This will be fixed in the next release.
`glide up` is a costly operation, which needs to be done only when adding new dependencies or updating dependencies to newer versions.